scripts/generate_lms.py

```diff
@@ -0,0 +1,100 @@
+from collections import defaultdict
+import sys
+import os
+import logging
+from generator_utils import load_config, common_args, update_model
+
+
+logger = logging.getLogger(__name__)
+
+
+def create_organizations(config):
```

Comment:
Please add docstring for this and all the functions that follows.

Reply:
Added. Sorry, I did rush to get the script working, so didn't have time to look at code quality. But now I've refactored and added docstring to everywhere applicable. 

Thanks for the suggestion. :)